pagopa/io-app
# feat(Payments): [#176176928] Add a switch to set payment method as favorite

## Short description
If the payment method is active, a switch mimicking the behaviour of the star with explicit explanation is shown.

## How to test
1. Create a new card
2. Set the new card as favorite using the switch
3. See the old favorite card no longer being marked as favorite


https://user-images.githubusercontent.com/26572302/107684891-9cbc7e80-6ca3-11eb-8bbb-5b2ff8a8f537.mov
